install/mysql/glpi-empty.sql

```diff
@@ -1940,6 +1951,7 @@ CREATE TABLE `glpi_deviceharddrives` (
   `entities_id` int unsigned NOT NULL DEFAULT '0',
   `is_recursive` tinyint NOT NULL DEFAULT '0',
   `deviceharddrivemodels_id` int unsigned DEFAULT NULL,
+  `deviceharddrivetypes_id` int unsigned DEFAULT NULL,
```

Comment:
Foreign keys is GLPI have usually `0` as default value, and cannot be nullable. This is uncommon, but it is preferable to follow this convention for now.
```suggestion
  `deviceharddrivetypes_id` int unsigned NOT NULL DEFAULT '0',
```